sdk/Sdk.Analyzers/IterableBindingTypeExpectedForBlobContainerPath.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.Azure.Functions.Worker.Sdk.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class IterableBindingTypeExpectedForBlobContainerPath : DiagnosticAnalyzer
```

Comment:
Should we move this analyzer and `WebJobsAttributesNotSupported.cs` to Sdk.Analyzers folder to keep them with all other analyzers?

Reply:
I'm not sure I follow; they are already in Sdk.Analyzers